jest.config.js

```diff
@@ -1,23 +1,27 @@
 module.exports = {
+  testEnvironment: 'jsdom',
   roots: [
     '<rootDir>/src',
     '<rootDir>/test'
   ],
   testMatch: [
-    '<rootDir>/test/**/*.(ts|js)'
+    '<rootDir>/test/**/*.(ts)'
```

Comment:
Here and below, I removed some things that aren't necessary for this repo, but if we'd rather have the config files be as consistent as possible between repos, that's totally fine by me.